### ISSUE TITLE
error message contrast issue

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1778]
From Digital Marin: https://digitalmarin-ca.proudcity.com/

I discovered a situation where there is insufficient contrast (to pass WCAG) on a form when there is an error after trying to submit and not filling in a required field.

To reproduce, go to any of the forms that have required fields and press the Submit button before filling in the fields. The fields with errors turn pink (background-color = #f2dede). Any text that was there or is typed in now has a red color = #b94a48. These two colors have a contrast ratio of only 3.94:1 as the Chrome Dev Tool shows in this screen shot:

![unnamed (2)](https://user-images.githubusercontent.com/6032426/99853300-b8a2ff00-2b50-11eb-82a9-4ee64dc82b40.jpg)

I noticed that elsewhere Proud City uses a slightly darker red color on the page which has compliant contrast. In the image above, the message below the input field which says “This field is required” uses color = #a94442, and that has a contrast ratio of 4.53:1 with the pink background and passes WCAG.

Can you request a change so that the darker red color is used for field contents too?

**Comment by aschmoe** [comment]
@lukefretwell @kevindherman  this will go out with release
